### PR TITLE
Updated install instructions

### PR DESCRIPTION
**Pull request by Shimwell**
Just adding the pip install to the README in preparation for the merge from develop to main, I also noticed both actions have the same name so it is difficult to know which is which.

### CODE REVIEW

**Comment by Shimwell** [comment]
Might want to hold of on merging this PR as it appears that the pip install package doesn't quite work over here. Might just be local issues but after running this command I'm not able to run the openmc-plotter command from the terminal (No such file or directory) so perhaps the script is not being copied to the correct folder during the install

```bash
pip install --index-url https://test.pypi.org/simple/ openmc-plotter
```

**Comment by pshriwise** [comment]
> Might want to hold of on merging this PR as it appears that the pip install package doesn't quite work over here. Might just be local issues but after running this command I'm not able to run the openmc-plotter command from the terminal (No such file or directory error) so perhaps the script is not being copied to the correct folder during the install

I just gave it a try and didn't run into any issues. :man_shrugging:

**Comment by Shimwell** [comment]
> > Might want to hold of on merging this PR as it appears that the pip install package doesn't quite work over here. Might just be local issues but after running this command I'm not able to run the openmc-plotter command from the terminal (No such file or directory error) so perhaps the script is not being copied to the correct folder during the install
> 
> I just gave it a try and didn't run into any issues. man_shrugging

must be my unique conda, python3, python2, environment which I think this captures quite well. https://xkcd.com/1987/

**Comment by paulromano** [comment]
FWIW, works for me!

**Comment by pshriwise** [comment]
@Shimwell did you have any new luck with that pip package using the method above? I've had success in a couple different places with it now.

**Comment by Shimwell** [comment]
I'm almost certain it is my local python. Please ignore my previous concerns. It works on my laptop but my main computer is not working. But I'm pretty sure I have a messed up environment.

File: README.md [hunk 1]
```diff
@@ -16,7 +18,7 @@ OpenMC, Matplotlib, NumPy, PySide2
 This package is most easily installed by running
 
 ```console
-pip install .
+pip install openmc-plotter
 ```
 
 from the top directory in the repository.
```
Comment (pshriwise) [inline README.md:24]: Looks like we can remove this line now?

**Review by pshriwise — CHANGES_REQUESTED** [review]
Just one tiny comment and I’m happy @Shimwell.

File: README.md (hunk 1 above)
Comment (Shimwell) [reply README.md:24]: I have removed ```pip install .``` and replaced it with ```pip install openmc-plotter``` in this PR should this be done differently

Comment (paulromano) [reply README.md:24]: I think @pshriwise's point was about the line below: "from the top directory in the repository" no longer makes sense.

Comment (Shimwell) [inline README.md:24]:
```suggestion
```

Comment (Shimwell) [reply README.md]: ah got you, I was distracted by the colors

**Review by pshriwise — APPROVED** [review]
Thanks @Shimwell! Love the self-suggestion to make the edit easier :ok_hand:

**Pull request merged**